src/fdk_rdf_parser/classes/dataservice.py

```diff
@@ -13,10 +13,10 @@ class DataService(PartialDcatResource):
 
     id: Optional[str] = None
     harvest: Optional[HarvestMetaData] = None
-    endpointDescription: Optional[List[str]] = None
-    endpointURL: Optional[List[str]] = None
+    endpointDescription: Optional[Set[str]] = None
+    endpointURL: Optional[Set[str]] = None
     mediaType: Optional[List[SkosCode]] = None
-    servesDataset: Optional[List[str]] = None
+    servesDataset: Optional[Set[str]] = None
     conformsTo: Optional[List[SkosConcept]] = None
```

Comment:
Hva med andre `List` felter. Er ikke det like greit å konvertere det til `Set` i samme slengen?

Reply:
Er det så mye å tjene på å skrive om til Set for objekt? Objektene blir vel tolket som ulike selv om de har de samme verdiene?

Reply:
Helt fair